mffpy/writer.py

```diff
@@ -57,6 +62,15 @@ def write(self):
             content.write(join(self.mffdir, filename), encoding='UTF-8',
                           xml_declaration=True, method='xml')
 
+            # remove from old files to overwrite
+            if filename in old_files:
+                idx = old_files.index(filename)
+                del old_files[idx]
+
+        # clean up in case of overwrite
+        for file in old_files:
+            remove(join(self.mffdir, file))
```

Comment:
In most cases this will work. However, sometimes the MFF directory contains subdirectories. For example, in `examples/example_4.mff`, there is a `subjects` folder with a couple files. We should add some logic to remove the file if it's a file or remove the directory and all contents if it's a directory. `shutil.rmtree` should do the trick.

Reply:
done